web/themes/joinup/templates/block/block--separated.html.twig

```diff
@@ -0,0 +1,45 @@
+{#
```

Comment:
There are added new classes and title has different tag, so there are several changes. But if needed I can move it to preprocess.